packages/ui/src/components/pagination/acvPagination.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts" setup>
+  import { computed } from 'vue';
+  import AcvPaginationItem from './acvPaginationItem.vue';
+  import type { AcvPaginationProps } from '@/components';
```

Comment:
You can put display name and description for pre-generated documentation here.

Reply:
added